examples.md

```diff
@@ -114,7 +118,9 @@ steps:
 ```yaml
 - uses: actions/cache@v2
   with:
-    path: ~/go/pkg/mod
+    path: |
+      ~/.cache/go-build
```

Comment:
I'm not familiar with Golang, but while googling this I came across [this cache example](https://github.com/mvdan/github-actions-golang#how-do-i-set-up-caching-between-builds) that shows different paths for different OS'es.  That link suggests this only works on Linux, correct?  Is there an env var or command that can be used to get the build cache path dynamically?

Reply:
🤦 

Oh @dhadka you're on the money 👍  - this is why we have +1's 😄 .

I'll update the PR now with a cross-os way of doing this 👍 . Sorry, I spend my life in Linux build agents typically.